src/pyodide/internal/setupPackages.ts

```diff
@@ -61,33 +67,38 @@ class SitePackagesDir {
    * If a file or directory already exists, an error is thrown.
    * @param {TarInfo} overlayInfo The directory that is to be "copied" into site-packages
    */
-  mountOverlay(overlayInfo: TarFSInfo): void {
+  mountOverlay(overlayInfo: TarFSInfo, dir: InstallDir): void {
```

Comment:
Can we make this a static method or top level function and directly pass `dest` as the argument?

Reply:
I guess we can, but I prefer this. Any reason you prefer a static method here?

Reply:
A method can access or modify anything in `this`, whereas a static method can't. So it's clearer to write static methods whenever it's possible.

Reply:
That's true, but I think it's less clean to expose the fields that are owned by ~~SitePackagesDir~~`VirtualizedDir`. If anything, I would refactor to make this class own the mounting logic so that the `TarFsInfo` fields are not public.